gta3_sbl/opcodes.txt

```diff
@@ -1,1224 +1,1196 @@
+{2239:} imgui_free_image {image} [int]
+{223A:} imgui_push_style_var {imGuiStyleVar} [ImGuiStyleVar] {val} [float]
+{223B:} imgui_push_style_var2 {imGuiStyleVar} [ImGuiStyleVar] {x} [float] {y} [float]
+{223C:} imgui_push_style_color {imGuiCol} [ImGuiCol] {r} [int] {g} [int] {b} [int] {a} [int]
+{223D:} imgui_pop_style_var {count} [int]
+{223E:} imgui_pop_style_color {count} [int]
+{223F:} [var drawList: int] = imgui_get_foreground_drawlist
+{2240:} [var drawList: int] = imgui_get_background_drawlist
+{2241:} [var drawList: int] = imgui_get_window_drawlist
+{2242:} imgui_drawlist_add_text {drawList} [int] {posX} [float] {posY} [float] {r} [int] {g} [int] {b} [int] {a} [int] {text} [string]
+{2243:} imgui_drawlist_add_line {drawList} [int] {p1X} [float] {p1Y} [float] {p2X} [float] {p2Y} [float] {r} [int] {g} [int] {b} [int] {a} [int] {thickness} [float]
+{2244:} [var fps: int] = get_framerate
+{2245:} [var version: string] = imgui_get_version
+{2246:} [var version: float] = imgui_get_plugin_version
+{2247:} imgui_set_cursor_visible {show} [bool]
+{2248:} [var height: float] = imgui_get_frame_height
+{2249:} [var x: float], [var y: float] = imgui_get_window_pos {uniqueId} [string]
+{224A:} [var width: float], [var height: float] = imgui_get_window_size {uniqueId} [string]
+{224B:} [var width: float], [var height: float] = imgui_calc_text_size {text} [string]
+{224C:} [var width: float] = imgui_get_window_content_region_width {uniqueId} [string]
+{224D:} [var x: float], [var y: float] = imgui_get_scaling_size {uniqueId} [string] {count} [int] {spacing} [bool]
+{224E:} [var width: float], [var height: float] = imgui_get_display_size
+{224F:} imgui_set_next_window_transparency {alpha} [float]
+{2250:} imgui_set_message {text} [string]
+{2251:} imgui_set_column_width {index} [int] {width} [float]
+{2252:} imgui_begin_childex {uniqueId} [string] {width} [float] {height} [float] {border} [bool] {flags} [int]
+{2253:} imgui_begin_disabled {disabled} [bool]
+{2254:} imgui_end_disabled
+{2255:}   imgui_begin_menu {label} [string] {enabled} [bool]
+{2256:} imgui_end_menu
+{0AF0:}   [var value: int] = read_int_from_ini_file {path} [string] {section} [string] {key} [string]
+{0AF1:}   write_int_to_ini_file {value} [int] {path} [string] {section} [string] {key} [string]
+{0AF2:}   [var value: float] = read_float_from_ini_file {path} [string] {section} [string] {key} [string]
+{0AF3:}   write_float_to_ini_file {value} [float] {path} [string] {section} [string] {key} [string]
+{0AF4:}   [var value: string] = read_string_from_ini_file {path} [string] {section} [string] {key} [string]
+{0AF5:}   write_string_to_ini_file {value} [string] {path} [string] {section} [string] {key} [string]
+{0BA2:}   [var handle: MemoryLibrary] = memory_load_dynamic_library {address} [int]
+{0BA3:} memory_free_dynamic_library [MemoryLibrary]
+{0BA4:}   [var address: int] = memory_get_dynamic_library_procedure {procName} [string] [MemoryLibrary]
```

Comment:
Why new is shorter?

Reply:
III/VC CLEO has commands with the same name but different ids (e.g. WRITE_MEMORY). Current script removes duplicates